schemaregistry/rest_service.go

```diff
@@ -251,28 +254,71 @@ func configureUSERINFOAuth(conf *Config, header http.Header) error {
 
 }
 
+func configureStaticTokenAuth(conf *Config, header http.Header) error {
+	bearerToken := conf.BearerAuthToken
+	if len(bearerToken) == 0 {
+		return fmt.Errorf("config bearer.auth.token must be specified when bearer.auth.credentials.source is" +
+			" specified with STATIC_TOKEN")
+	}
+	header.Add("Authorization", fmt.Sprintf("Bearer %s", bearerToken))
+	setBearerAuthExtraHeaders(conf, header)
+	return nil
+}
+
+func setBearerAuthExtraHeaders(conf *Config, header http.Header) {
+	targetIdentityPoolID := conf.BearerAuthIdentityPoolID
+	if len(targetIdentityPoolID) > 0 {
```

Comment:
should we throw an error if len == 0 since `BearerAuthIdentityPoolID` is a required config?

Reply:
targetIdentityPoolID is only required by cloud SR. Let me add a comment.